recipes/g3log/config.yml

```diff
@@ -0,0 +1,8 @@
+versions:
+  "1.3.2-86":
+    folder: all
+  "1.3.2":
+    folder: all
+  "20170108":
```

Comment:
Please only add date versions when they project does no formal releases at all.

If you need a new version please ask first upstream for a new one.

Mixing date versions and actual versions numbers would be very confusing

Reply:
OK, I can remove the date version.

Reply:
https://github.com/conan-io/conan-center-index/wiki/FAQ#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
Fixed in 0585deb

Reply:
Sorry, wrong commit reference. Fixed in ffa6445